src/main/java/io/vertx/core/net/TCPSSLOptions.java

```diff
@@ -102,6 +102,11 @@ public abstract class TCPSSLOptions extends NetworkOptions {
    */
   public static final boolean DEFAULT_TCP_QUICKACK = false;
 
+  /**
+   * The default TCP_USER_TIMEOUT value in milliseconds = null
+   */
+  public static final Integer DEFAULT_TCP_USER_TIMEOUT = null;
```

Comment:
For consistency, can you please change this to `int` and use the default value of `0`?

Reply:
As you change it, can you please verify your git configuration for author and user emails? Make sure to sign off the commit and that you signed the Eclipse Contributor Agreement using the same address.

Reply:
Hi @tsegismont, thanks for the quick feedback. I'll do the necessary changes.

Reply:
Hi @tsegismont, I've fixed the default value to '0' and signed both commits and Eclipse Contributor Agreement. Thanks.

Reply:
I think that when the value is > 0 then we should warn on the console that the user timeout will not be honoured when it is not epoll

Reply:
Hi @tsegismont and @vietj. Could you point me to where should I add this warning please? Also there are other options that would only work with Linux native: SO_REUSEPORT, TCP_QUICKACK, TCP_CORK and TCP_FASTOPEN, should I also add a warning for them?

I noticed that the tests for Ubuntu-latest, 8, with profile `testDomainSockets` has failed, I'm not sure why, as I run it in my local ubuntu and they passed.

Thanks.

Reply:
To me it's fine without a warning because we don't do it for other props that are applied to native setup only.

It could be addressed in a follow-up PR if that's an important concern.

Reply:
I've restarted the failed jobs, I don't believe the failure is related.

Reply:
Thanks guys, I've added this info in the TCPSSLOptions setter: https://github.com/eclipse-vertx/vert.x/blob/master/src/main/java/io/vertx/core/net/TCPSSLOptions.java#L770

Please let me know if another PR is required for the javadocs.

Cheers.